components/bl-multiple-select-component/src/index.js

```diff
@@ -7,15 +7,19 @@ import { useOnClickOutside, validate } from './helpers';
 const { cn } = BackendlessUI.CSSUtils;
 
 export default function MultipleSelectComponent({ component, eventHandlers }) {
-  const { display, classList, disable, placeholder, variant, type, value, options } = component;
+  const { display, classList, disabled, placeholder, variant, type, value, options } = component;
   const { onChange } = eventHandlers;
-  
+
   const rootRef = useRef(null);
   const [isOptionsOpen, setIsOptionsOpen]= useState(false);
   const [selectValue, setSelectValue] = useState([]);
   const [isSelectActive, setIsSelectActive] = useState(false);
-  
+
   const optionsList = useMemo(() => validate(options), [options]);
+  const classes = cn(
+    'bl-customComponent-multipleSelect', variant, classList,
+    { "bl-customComponent-multipleSelect--disabled": disabled }
```

Comment:
single quotes must be everywhere

editorconfig must fix it, tell me if it isn't true and we will try to add precommit hooks